wpilibNewCommands/src/main/native/include/frc2/command/CommandRobot.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <frc/TimedRobot.h>
+#include <units/math.h>
+#include <units/time.h>
+
+namespace frc2 {
+
+/**
+ * The CommandRobot class extends the {@link TimedRobot} class in order to ease
+ * functionality for Command Based Programming.
+ */
+class CommandRobot : public frc::TimedRobot {
```

Comment:
In RobotPy, we have a similar thing and called it [TimedCommandRobot](https://github.com/robotpy/robotpy-commands-v2/blob/main/commands2/src/TimedCommandRobot.h). Should we call it the same here?

Reply:
I agree that TimedCommandRobot is a better name for this.